src/components/general/HtmlDisplay.vue

```diff
@@ -5,26 +5,18 @@
 
 <script setup lang="ts">
 import { getResourcesStaticUrl } from '@/lib/fews-config'
-import { useConfigStore } from '@/stores/config'
 import { computedAsync } from '@vueuse/core'
-import { computed } from 'vue'
-import { useRoute } from 'vue-router'
 
-const route = useRoute()
+interface Props {
+  url: string | undefined
+}
 
-const configStore = useConfigStore()
-const matchingComponent = computed(() => {
```

Comment:
What did this used to do? Was this just dead code?

Reply:
Moved to the HtmlDisplayView, small refactor to allow for custom url without the dependency on the page route